### PR TITLE
Fix the schedule navigation touchables not working on Android

### PR DESCRIPTION
**Pull request by mikeverf**
> Fix the schedule navigation touchables not working on Android

This PR closes ticket #68

### CODE REVIEW

**Comment by devopsITP** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-68-fix_touchables_android

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by devopsITP** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-68-fix_touchables_android

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

File: components/SchedulePagination.js [hunk 1]
```diff
@@ -34,24 +34,24 @@ const styles = StyleSheet.create({
...
-    <InvisibleNavButtons index={index} total={total} onNextTap={onNextTap} />
-  </View>
+  <TouchableWithoutFeedback onLongPress={onLongPress}>
+    <View style={styles.container}>
+      {tracks[index - 1] ? (
+        <NavigationItem title={tracks[index - 1].title} position="left" />
+      ) : (
+        <NavigationItem position="left" />
+      )}
+      {/* <TouchableWithoutFeedback onLongPress={onLongPress}> */}
```
Comment (thibmaek) [inline components/SchedulePagination.js]: comment code

**Comment by devopsITP** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-68-fix_touchables_android

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Pull request merged**